backend/.env.docker

```
@@ -1,10 +1,10 @@
-MONGODB_URI="mongodb://mongo-service/wanderlust"
+MONGODB_URI="mongodb://mongo-service:27017/wanderlust"
 REDIS_URL="redis://redis-service:6379"
 PORT=8080
-FRONTEND_URL="http://34.213.249.66:5173"
+FRONTEND_URL="http://34.213.249.66:31000"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)